src/lib.rs

```diff
@@ -21,12 +21,58 @@ use std::fmt::Write;
 use tree_sitter::{Language, Node};
 
 pub struct FormatOptions {
+    /// Number of spaces used for one level of indentation
     pub indentation: usize,
+    /// Wether to align prefix IRIs,
+    /// or use a single space after the ':'.
+    pub align_prefix_iris: bool,
+    /// Wether to re-indent comments,
+    /// ensuring they all have exactly one space after the '#'.
+    pub unify_comment_indents: bool,
```

Comment:
we can maybe just always do that

Reply:
I don't like this.
This makes sense for text comments, but when the comment is actually commented-out "code", then this is very-much unwanted.

Reply:
... thanks for the reviewing! :-)

Reply:
Edit: it might not be a good idea in case of multi-line comments where the author might want some indentation.

Reply:
Edit: it might not be a good idea in case of multi-line comments where the author might want some indentation.

Reply:
Agreed, maybe let's skip this option?

Reply:
hmm.. at first though that makes sense - and it does, as the inline comments are less likely to be code - but could be code too, and it might also be confusing to users of the tool if most comments stay the same, but inline ones are changed. But I am soft on this one.. no strong feeling about it.

Reply:
Make sense! Let's not touch comments. Thank you!